save-cloud-common/src/jvmMain/kotlin/org/cqfn/save/entities/Execution.kt

```diff
@@ -48,11 +52,17 @@ class Execution(
     @Enumerated(EnumType.STRING)
     var type: ExecutionType,
 
-    var version: String
+    var version: String,
+
+    var passedTests: Long,
+
+    var failedTests: Long,
+
+    var skippedTests: Long,
 
 ) : BaseEntity() {
     /**
      * @return Execution dto
      */
-    fun toDto() = ExecutionDto(status, type, version)
+    fun toDto() = ExecutionDto(status, type, version, endTime?.format(DateTimeFormatter.ofPattern("dd-MM-yyyy HH:mm:ss")), passedTests, failedTests, skippedTests)
```

Comment:
Why do we have endTime as String in DTO? I would expect it to be Long timestamp, or some serializable class from kx.datetime

Reply:
Because we use it only in front-end and I thought, to avoid problem with cast use `endTime` as `String`

Reply:
Usually timestamp is a more universal representation, which will be converted to the desired format on frontend without the need to parse string in another format

Reply:
BTW, we already have `org.cqfn.save.utils.LocalDateTimeSerializer` to make it easier to convert `LocalDateTime` to timestamp.